src/com/google/javascript/jscomp/DefaultPassConfig.java

```diff
@@ -174,6 +174,17 @@ void maybeInitializePreprocessorSymbolTable(AbstractCompiler compiler) {
       }
     }
   }
+  
+  @Override
+  protected List<PassFactory> getAlwaysPreChecks() {
+    List<PassFactory> preChecks = new ArrayList<>();
+    
+    if (options.wrapGoogModulesForWhitespaceOnly) {
```

Comment:
Seems a strange to me that wrapGoogModulesForWhitespaceOnly changes the behavior for non-whitespace only code.
My inclination is to just make this the default for whitespace-only when closurePass is set, and remove the option.